hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/config/internal/KeyValuePairConverter.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.spi.config.internal;
+
+import com.hedera.node.app.spi.config.types.KeyValuePair;
+import com.swirlds.config.api.converter.ConfigConverter;
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+/**
+ * Implementation of {@link ConfigConverter} that supports {@link KeyValuePair} as data type for the config.
+ */
+public class KeyValuePairConverter implements ConfigConverter<KeyValuePair> {
+
+    private static final String PATTERN = Pattern.quote(";");
```

Comment:
I wonder, if we should use ':' or '=' instead. Seems more common in similar cases.

Reply:
Totally fine with any of them.